packages/unlock-js/src/UnlockSwapPurchaserABI.ts

```diff
@@ -0,0 +1,3 @@
+export const UnlockSwapPurchaserABI = [
```

Comment:
Please check where/if we already have the ABI for ERC20 and use the same folder?